plugins/rikki/heroeslounge/components/ScheduleMatch.php

```diff
@@ -69,21 +64,20 @@ public function onSaveDate()
         $match = Match::find(post('match_id'));
         if ($match) {
             $date = post('date');
-            $timezone = post('timezoneName');
+            $timezone = TimezoneHelper::getTimezone();
             if ($date != null) {
                 try {
                     $x = new DateTime($date, new DateTimeZone($timezone));
-                    $x->setTimezone(new DateTimeZone('Europe/Berlin'));
+                    $x->setTimezone(new DateTimeZone(TimezoneHelper::defaultTimezone()));
                     $match->wbp = $x->format('Y-m-d H:i:s');
                     if ($match->tbp != null && Carbon::parse($match->wbp) < Carbon::parse($match->tbp)) {
                         $match->save();
                         Flash::success('Match has been successfully scheduled for '.$date);
                     } else {
-                        $y = new DateTime($match->tbp, new DateTimeZone('Europe/Berlin'));
+                        $y = new DateTime($match->tbp, new DateTimeZone(TimezoneHelper::defaultTimezone()));
```

Comment:
Also UTC

Reply:
Same as above: it is, if that is properly configed, which you said it is.